appveyor.yml

```diff
@@ -0,0 +1,41 @@
+version: "{build}"
+
+# Init is run before the repo is cloned
+# Windows line endings.
+init:
```

Comment:
In my opinion, some of the comments in this file are rather superfluous. I mean, a matrix of `nodejs_versions` with a comment of `Node version to test against` is rather duplicated 😬